docs/policy/community-testing.md

```diff
@@ -29,20 +29,22 @@ After users have been notified of changes, minor packages will be marked eligibl
 negative feedback after 7 calendar days.
 In addition to the above requirements, major packages must also receive positive feedback and be approved by the Release
 Manager.
+If a major package's functionality is tested by the OSG VM Universe tests, the release manager may approve that
+package for release without positive feedback after the package has been in testing for four weeks.
```

Comment:
```suggestion
If a major package has not received feedback after four weeks and it has been sufficiently tested by OSG integration
tests, the Release Manager may approve the package for release.
```

I think jamming the two conditions together in a single clause makes it easier to understand